src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/Wallet.java

```diff
@@ -141,6 +141,12 @@ public RegistrationStatusResponse migrateAccounts(List<MigrationAccount> account
         if (this.rpc == null) {
             throw new NoSuchFieldException("Before using migrateAccounts, rpc must be set. You should call initNodeAPI first.");
         }
+        if (this.rpc.getWeb3jService() instanceof HttpService) {
```

Comment:
Don't we need to consider condition if the Node API has initialized with websocket?

Reply:
It would be great to add that condition too 🙂 .
I'll add the condition for `WebsocketService`. Thanks for your feedback.